processor/runtime/src/main/resources/application.properties

```diff
@@ -1,3 +1,8 @@
 # Configuring the outgoing channel (writing to Kafka) for Actions
 mp.messaging.outgoing.actions-out.connector=smallrye-kafka
 mp.messaging.outgoing.actions-out.value.serializer=org.apache.kafka.common.serialization.StringSerializer
+
+mp.messaging.outgoing.actions-out.bootstrap.servers=${KAFKA_BOOTSTRAP_ACTIONS_OUT_SERVERS:localhost:9092}
+mp.messaging.outgoing.actions-out.sasl.mechanism=PLAIN
+mp.messaging.outgoing.actions-out.security.protocol=${KAFKA_SECURITY_PROTOCOL:SASL_SSL}
+mp.messaging.outgoing.actions-out.sasl.jaas.config=org.apache.kafka.common.security.plain.PlainLoginModule required username="${KAFKA_CLIENT_ACTIONS_OUT_ID:example-kafka-client-actions-out-id}" password="${KAFKA_CLIENT_ACTIONS_OUT_SECRET:example-kafka-client-actions-out-secret}";
```

Comment:
As far as I see here we have a default `kafka` configuration [here](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/executor/src/main/resources/application.properties#L6-L10) that is probably "inherited" (as far as I understand) by [`mp.messaging.incoming.events-in`](https://github.com/5733d9e2be6485d52ffa08870cabdee0/sandbox/blob/main/executor/src/main/resources/application.properties#L17).

What about unify? Can you please update `mp.messing.incoming` to directly configure these properties?